src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,36 @@
+package duke.tasks;
+
+public class Task {
+    protected String taskName;
+    protected Boolean status;
+
+    public Task(String taskName) {
+        this.taskName = taskName;
+        this.status = false;
+    }
+
+    public String getTaskName() {
+        return taskName;
+    }
+
+    public String getStatus() {
+        if (status) {
+            return "[X]";
+        } else {
+            return "[ ]";
+        }
+    }
+
+    public void setDone() {
+        this.status = true;
+    }
+
+    @Override
+    public String toString() {
+        if (status) {
+            return "[X] " + getTaskName();
+        } else {
+            return "[ ] " + getTaskName();
+        }
+    }
+}
```

Comment:
Maybe you can name boolean variable status as isDone!

Reply:
Thank you for the suggestion, will defo consider it